src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ConversionCompletionProviderTests.cs

```diff
@@ -0,0 +1,797 @@
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Completion.Providers;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Completion;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Newtonsoft.Json.Linq;
+using System.Collections;
+using System.Numerics;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders
+{
+    public class ConversionCompletionProviderTests : AbstractCSharpCompletionProviderTests
+    {
+        public ConversionCompletionProviderTests(CSharpTestWorkspaceFixture workspaceFixture) : base(workspaceFixture)
+        {
+        }
+
+        internal override Type GetCompletionProviderType()
+            => typeof(ConversionCompletionProvider);
+
+        protected override string? ItemPartiallyWritten(string? expectedItemOrNull) =>
+            expectedItemOrNull switch
+            {
+                { Length: >= 2 } s when s.StartsWith("(") => expectedItemOrNull.Substring(1, 1),
+                _ => base.ItemPartiallyWritten(expectedItemOrNull)
+            };
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(47511, "https://github.com/dotnet/roslyn/issues/47511")]
+        public async Task OperatorIndexerCompletionItemsShouldBePlacedLastInCompletionList()
+        {
+            var castCompletionItem = (await GetCompletionItemsAsync(@"
+public class C
+{
+    public static explicit operator float(C c) => 0;
+}
+
+public class Program
+{
+    public void Main()
+    {
+        var c = new C();
+        c.$$
+    }
+}
+", SourceCodeKind.Regular)).Single();
+
+            var completionList = new[] {
+                CompletionItem.Create("SomeText0"),
+                castCompletionItem,
+                CompletionItem.Create("SomeText1"),
+                CompletionItem.Create("\uffdcStartWith_FFDC_Identifier"), // http://www.fileformat.info/info/unicode/char/ffdc/index.htm
+                CompletionItem.Create("SomeText2"),
+                CompletionItem.Create("\uD884\uDF4AStartWith_3134A_Identifier"), // http://www.fileformat.info/info/unicode/char/3134a/index.htm
+                CompletionItem.Create("SomeText3"),
+            };
+            Array.Sort(completionList);
+            Assert.Collection(completionList,
+                c => Assert.Equal("SomeText0", c.DisplayText),
+                c => Assert.Equal("SomeText1", c.DisplayText),
+                c => Assert.Equal("SomeText2", c.DisplayText),
+                c => Assert.Equal("SomeText3", c.DisplayText),
+                c => Assert.Equal("\uD884\uDF4AStartWith_3134A_Identifier", c.DisplayText),
```

Comment:
having bizarre characters in the DisplayText feels very rwong to me.  This should be in .SortText.  The first is expected to be normal bits of text (that could even be consumed by other peple calling teh api).  .SortText is there to control where things go in the list.

Reply:
(i think we may also have a Priority/Sorting category for this purpose as well.

Reply:
The bizarre symbols are valid C# identifier start characters. So a method name could start with these.